src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$accent-color: #00acdc;
+
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+h2,
+h3,
+p {
+  margin: 0;
+}
```

Comment:
these styles will be better to move in other file because it's not related to `card` block